### PR TITLE
Fix renaming of agenda items with proposal

### PR DESCRIPTION
**Pull request by tarnap**
change_agenda_item_title_fixed

Resolves https://github.com/4teamwork/gever/issues/160

### CODE REVIEW

File: opengever/meeting/model/agendaitem.py [hunk 1]
```diff
@@ -211,6 +211,7 @@ def get_title(self, include_number=False):
     def set_title(self, title):
         if self.has_proposal:
             self.submitted_proposal.title = title
+            self.submitted_proposal.sync_model()
```
Comment (deiferni) [inline opengever/meeting/model/agendaitem.py]: could you add a test for this, or extend an existing test and make sure the title is set correctly on sql and plone?

Comment (tarnap) [reply opengever/meeting/model/agendaitem.py]: Of course! :)

**Comment by tarnap** [comment]
@deiferni: This is probably the kind of test you wanted... but, could you suggest a better title for the created `agenda_item`? https://github.com/4teamwork/opengever.core/pull/3639/commits/4d7f0afe01e70841ad57156bfcf33656bf68ea61

**Comment by tarnap** [comment]
@deiferni: shall I squash the commits together before merging?

**Comment by deiferni** [comment]
@tarnap i'd say so, looks small enough for one commit.

**Pull request merged**